components/presenters/ApiTable/adapters/ApiTableAdapter.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import camelCase from 'lodash/camelCase'
+
+import { ComponentWithClass } from '../../../../common/ComponentWithClass'
+import { ContentPanel, LeftCol, RightCol } from '../../ContentPanel'
+import {
+  ApiTable,
+  ApiTableItem,
+  ApiTableDefaultValue,
+  ApiTableDescription,
+} from '..'
+
+interface ApiTableAdapterProps extends ComponentWithClass {
+  fields: any
+}
+
+export const ApiTableAdapter: React.FC<ApiTableAdapterProps> = ({ fields }) => {
+  const { title, apiTableDescription, apiFieldCollection } = fields
+  const id = camelCase(title)
+
+  return (
+    <ContentPanel id={id}>
+      <LeftCol>
+        {title && <h2>{title}</h2>}
+        {apiTableDescription}
+        <ApiTable>
+          {apiFieldCollection &&
+            apiFieldCollection.items.map(
+              ({ dataType, defaultValue, description, name, required }) => {
```

Comment:
Could we do something with the query so that the names of these match the props on the `ApiTableItem`? Might make it easier to read..